Beams/MRML/vtkMRMLRTBeamNode.cxx

```diff
@@ -912,3 +916,9 @@ void vtkMRMLRTBeamNode::CreateMLCPointsFromSectionBorder( double jawBegin,
   }
   side12.push_back(side2.back());
 }
+
+//----------------------------------------------------------------------------
+bool vtkMRMLRTBeamNode::AreEqual( double v1, double v2)
+{
+  return vtkSlicerRtCommon::AreEqualWithTolerance( v1, v2);

```

Comment:
This alias function seems unnecessary, it saves a few characters where this needs to be called, but makes the API mor complex.